libraries/AP_NavEKF2/AP_NavEKF2_core.h

```diff
@@ -329,7 +336,19 @@ class NavEKF2_core : public NavEKF_core_common
     // return true when external nav data is also being used as a yaw observation
     bool isExtNavUsedForYaw(void);
 
+    // get solution data for the EKF-GSF emergency yaw estimator
+    // return false if data not available
+    bool getDataEKFGSF(float *yaw_composite, float *yaw_composite_variance, float yaw[N_MODELS_EKFGSF], float innov_VN[N_MODELS_EKFGSF], float innov_VE[N_MODELS_EKFGSF], float weight[N_MODELS_EKFGSF]);
```

Comment:
I hear from Tridge that this method may disappear but in general it's safer (and easier) to pass by reference rather than using pointers.  If you just change the argument to look like this:

>> bool getDataEKFGSF(float &yaw_composite, float &yaw_composite_variance, ...

Then inside the method you can just use the variables like regular variables.. assign them to new values and the passed in variable will change.